src/grimp/application/usecases.py

```diff
@@ -137,11 +144,23 @@ def _scan_packages(
             else:
                 imports_by_module_file[module_file] = direct_imports
 
-    # TODO Parallelise this part.
-    for module_file in modules_files_to_scan.difference(imports_by_module_file):
-        imports_by_module_file[module_file] = import_scanner.scan_for_imports(
-            module_file.module, exclude_type_checking_imports=exclude_type_checking_imports
+    remaining_modules_files_to_scan = list(
```

Comment:
I think the `_scan_packages` function has tipped over into becoming too long now - would you mind breaking it up a bit?